api_v3/services/MediaService.php

```diff
@@ -1090,10 +1090,19 @@ protected function prepareEntryForInsert(KalturaBaseEntry $entry, entry $dbEntry
 		$entry->validatePropertyNotNull("mediaType");
 
 		$conversionQuality = $this->getConversionQuality($entry);
-		if (!is_null($conversionQuality))
+		if (!is_null($conversionQuality)) {
 			$entry->conversionQuality = $conversionQuality;
+			if (!$entry->conversionProfileId) {
+				$entry->conversionProfileId = $entry->conversionQuality;
+			}
+		}
+
+		if ($dbEntry == null){
+			$dbEntry = $this->duplicateTemplateEntry($entry->conversionProfileId);
```

Comment:
Created https://kaltura.atlassian.net/browse/PLAT-4073 to give this issue the attention needed